sdk/src/main/scala/coop/rchain/sdk/primitive/VoidSyntax.scala

```diff
@@ -0,0 +1,13 @@
+package coop.rchain.sdk.primitive
+
+trait VoidSyntax {
+  implicit def sdkSyntaxVoid[A](a: A): VoidOps[A] = new VoidOps(a)
+}
+
+final class VoidOps[A](private val a: A) extends AnyVal {
+
+  /**
+    * Ignore value and return [[Unit]].
+    */
+  def void(): Unit = ()
```

Comment:
I thought the parentheses were redundant and removed them, but got a warning.

![image](https://user-images.githubusercontent.com/1443855/176641158-f8f00d25-4729-4a9d-a6ba-16aea0012fb8.png)

What does it mean? :thinking: